schedule/yast/rmt_server_install/rmt_server_install.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: rmt_server_install
+description: |
+  Perform an installation enabling some modules and selecting some
```

Comment:
could we improve this description and describe the test case, installation using registration server RMT trusting and importing blablab and validation of ... for example.

Reply:
Done!